common/src/main/java/com/revenuecat/purchases/common/HTTPClient.kt

```diff
@@ -175,10 +175,9 @@ class HTTPClient(
         }
 
         val verificationResult = if (shouldSignResponse &&
-            nonce != null &&
```

Comment:
Currently nonce will never be null if `shouldSignResponse`, but that will change with the new static endpoints we want to sign.

Reply:
A separate PR for that is fine 👍🏻